pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "generateBuildrootSBOM"
+version = "1.0.8"
```

Comment:
I cannot tell how to set the project version at all. I have versions in github and versions in pypi. What version value should be used here since this is for the packaging concept? Based on what you suggest I should use my CycloneDX-Buildroot version value which seems correct. Can update.

Reply:
yes, the value you use here will directly be influencing the version used in pypi.